mmdeploy/backend/onnxruntime/wrapper.py

```diff
@@ -36,7 +38,8 @@ class ORTWrapper(BaseWrapper):
     def __init__(self,
                  onnx_file: str,
                  device: str,
-                 output_names: Optional[Sequence[str]] = None):
+                 output_names: Optional[Sequence[str]] = None,
+                 enable_trt: bool = False):
```

Comment:
Is it possible to check if TensorRT provider is available in the current environment?
If it is, we can check it inside init instead of add a flag.

Reply:
```
>>> available_providers = onnxruntime.get_available_providers()
>>> available_providers
['TensorrtExecutionProvider', 'CUDAExecutionProvider', 'CPUExecutionProvider']
```

`onnxruntime` has a `get_available_providers` function that tells you which providers are available in the current environment. Changed to use TensorRT executor if available. thank you :)